### ISSUE TITLE
Help building examples

### ISSUE THREAD

**Issue by vanfanel** [michalsc/Emu68#131]
Hi there, @michalsc 

I am trying to give a go at EMU68.
I have succesfully built latest EMU68 code for standalone Pi, but building the examples to give it something to run is resisting...
```

manuel@hp15db0:~/Emu68/examples$ make
make[1]: Entering directory '/home/manuel/Emu68/examples/SmallPT'
Building target: ../../Build/SmallPT
/opt/amiga/lib/gcc/m68k-amigaos/6.5.0b/../../../../m68k-amigaos/bin/ld: cannot open output file ../../Build/SmallPT: No such file or directory
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:11: ../../Build/SmallPT] Error 1
make[1]: Leaving directory '/home/manuel/Emu68/examples/SmallPT'
make: *** [Makefile:13: SmallPT] Error 2
```

I also installed `gcc-m68k-amigaos` from this repo:
https://github.com/bebbo/amiga-gcc

So, any idea on what am I missing?

Thanks!

**Comment by michalsc** [comment]
Yes, the Build directory in the root of the source tree is missing. The examples are using my old version of makefiles which cannot create that directory.

Short "fix" would be something like
```shell
mkdir Build
cd examples
make```

**Comment by vanfanel** [comment]
@michalsc Thanks for your answer!

I am able to build them by now, but all I get (in standalone mode, no PiStorm) is a black screen on the PI4 HDMI port.

So, are those supposed to be visible by now in standalone mode?

**Comment by vanfanel** [comment]
@michalsc Hello,

Is aros68k runnable with emu68? I understand a special build of aros68k would be needed... Is it available somewhere?

**Comment by vanfanel** [comment]
Ehmm... why the long silence on this?
Why confine Emu68 to serve old hardware when it can be the base for new hardware?

**Comment by michalsc** [comment]
Sorry, was busy with life and with WiFi for Emu68 - the latter takes a lot of time due to total lack of documentation. Regarding old hardware and new hardware - we are working on both.

**Comment by vanfanel** [comment]
Thanks for the answer, Michal!

Do you mean you are also working on standalone Emu68? Is there an AROS 68K build I can run with the standalone version?

**Comment by michalsc** [comment]
Not at the moment. But I and PiStorm team are looking for new/moderner hardware platform for Emu68. AROS native? Yes, definitely at some point, especially since I will be able to re-use all the drivers I wrote so far for emu68 (they are all m68k)

**Comment by JJDSNT** [comment]
It is great all the movement around, i got lost of how things will be.
Will aros run on emu68?
I saw claude with a tang nano 20k, i am wondering if that will become a new minimig (not sure if it will fit), or how erverything will glue togheter.
I look for it all and have the intuition that one day all will be glued together (tang nano 20k, raspberry, aros and emu68), i just dont know exactly how.

Off-topic, just out of curiosity: risc-v will be of any good for the emu68 project? (im allready very happy with the idea of tang nano 20k+raspberry and all results this effort can produce (as in arm land, for example), but i am not sure if risc-v is big endian enough for emu68)

**Comment by vanfanel** [comment]
@michalsc Are there any news with regard to standalone AROS on EMU68?
I know you had some interest on standalone a couple of years ago, and in fact your own drivers for AROS would be very useful on standalone EMU68.

**Comment by michalsc** [comment]
Closing old issue (which is not an issue actually).
